app/models/legal_aid_application.rb

```diff
@@ -157,6 +157,10 @@ def application_proceedings_by_name
     types.sort_by(&:meaning)
   end
 
+  def section_8_proceedings?
+    proceeding_types.any? { |type| type.ccms_matter == 'Section 8 orders' }
```

Comment:
I have something similar in my task list PR, how would you feel about using the ccms_matter_code, I felt that it had less scope for typos, e.g. 
``` ruby
type.ccms_matter.eql?('KSEC8')
```
String comparison isn't great (but all either of us could use) and `Section 8 orders` could be `section 8 order` or whatever sausage finger dev types in the future 😄

Reply:
Ah yes, this makes sense, I'll update it!